webapp/src/simulator.ts

```diff
@@ -329,7 +329,8 @@ export function run(pkg: pxt.MainPackage, debug: boolean,
         autoRun,
         ipc: isIpcRenderer,
         dependencies,
-        activePlayer: playerNumber
+        activePlayer: playerNumber,
+        theme: pkg.config.theme,
```

Comment:
Should we have the field in the config be `simTheme` rather than just `theme` since it only applies to the simulator?

Reply:
Since it's in pxtJson it feels similar level to me as  `palette` in that it's assumed it applies to the output for the project (simulator/hw), but if it feels confusing we can change -- only thing is that the snippet that goes in the markdown would also change to `{ "simTheme": "bubblegum" }` which is a little more room for error (e.g. not camel casing, etc)